examples/attach.py

```diff
@@ -1,16 +1,16 @@
 import asyncio
 import sys
 
-from golem_core.core.activity_api import commands
-from golem_core.core.golem_node import GolemNode
-from golem_core.core.payment_api import DebitNote
-from golem_core.core.resources import NewResource
+from golem.node import GolemNode
+from golem.resources import DebitNote, NewResource
```

Comment:
this is most likely completely out of scope of this review but, for the sake of readability and consistency, shouldn't events be named in some standardised way? e.g. through a pair of Noun+Action, e.g. `ResourceCreated` instead of just Adjective+Noun which suggests rather a data object than an event ?...

Reply:
Added to our workllist